src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -167,6 +167,20 @@ void saveAndLoadHugeFiles(@TempDir final Path tmp) throws Exception {
         );
     }
 
+    @Test
+    void shouldReturnContentWithSpecifiedSize() throws Exception {
+        final byte[] content = "1234".getBytes();
+        final Key key = new Key.From("key");
+        this.storage.save(
+            key,
+            new Content.OneTime(new Content.From(content))
+        ).get();
+        MatcherAssert.assertThat(
+            this.storage.value(key).get().size().get(),
+            new IsEqual<>((long) content.length)
+        );
+    }
+
```

Comment:
@genryxy this test could be done for all `Storage` and in my opinion it will be right to do so. Please move it to `StorageSaveAndLoadTest` so we will ensure that all `Storage` implementation return `Content` with size